data/config/presale.rinkeby.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  infra: {
+    ens: '0x98df287b6c145399aaa709692c8d308357bc085d',
+    daoFactory: '0xad4d106b43b480faa3ef7f98464ffc27fc1faa96',
+    registrar: '0x3665e7bfd4d3254ae7796779800f5b603c43c60d'
+  },
+  instance: {
+    owner: '0x9aedfbe0f3ed33cbb71f937cb9eb0ff1f4dfc076',           // AN-DAO controller
+    id: 'anj-presale',
+    collateralToken: '0xF68eb8077744b49F2B289c0Fd12Bbc9f3729dFe3', // ANT
+    bondedToken: '0x1B1EdCF4f3C5bB705a7670154e2C65569d0676Be',     // ANJ
+    period: '86400',                                               // 1 day
+    exchangeRate: '100000000',                                     // 1/100
+    openDate: '0',
```

Comment:
Let's open it immediately, same here about the comments on each variable. Let's describe clearly what the mean, as we did for the court

Reply:
We cannot set a date to open immediately, as we don't know when this will be executed.

Reply:
Sorry, I was wrong. I'll fix it.